subprojects/test-runner/device-provider/src/test/kotlin/com/avito/android/runner/devices/internal/kubernetes/KubernetesReservationClientTest.kt

```diff
@@ -152,8 +155,29 @@ internal class KubernetesReservationClientTest {
         }
         runBlocking {
             client.claim(listOf(ReservationData.stub()), this)
-            delay(100) // wait inner parts of `claim` fun
             client.release()
         }
     }
+
+    @Test
+    fun `can't boot device and delete device fail - throws exception`() {
```

Comment:
```suggestion
    fun `can't boot device and then delete device fail - throws exception`() {
```

adding `then` indicates that there are two operations in order, previous variant reads as there are two independent operation that both leads to an exception